dotnet/requestsApp/SaltEdge.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Security.Cryptography;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace SESample
+{
+    class SaltEdge
+    {
+        const int REQUEST_EXPIRES_MINUTES = 3;
+        const string APP_ID = "123";
+        const string SECRET = "123";
```

Comment:
Maybe extract this into a config file? And raise error/ add warning if developers don't configure it properly?

Reply:
implemented